api/src/main/java/org/openmrs/module/aijar/api/impl/AijarServiceImpl.java

```diff
@@ -302,4 +302,35 @@ private String replaceLettersWithNumber(String letter) {
 		}
 		return numberToReturn;
 	}
+	/**
+	 * @see org.openmrs.module.aijar.api.AijarService#stopActiveOutPatientVisits()
+	 */
+	public void stopActiveOutPatientVisits() {
+
+		SimpleDateFormat formatter = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
+
+		SimpleDateFormat formatterExt = new SimpleDateFormat("yyyy-MM-dd");
+
+		String formattedDate = formatterExt.format(new Date()) + " " + "00:00:00";
+
+		AdministrationService administrationService = Context.getAdministrationService();
+
+		String visitTypeUUID =administrationService.getGlobalProperty("ugandaemr.autoCloseVisit.visitTypeUUID");
+
+		VisitService visitService = Context.getVisitService();
+
+		List activeVisitList = null;
+		activeVisitList = administrationService.executeSQL("select visit.visit_id from visit inner join visit_type on (visit.visit_type_id = visit_type.visit_type_id)  where visit_type.uuid='"+visitTypeUUID+"' AND visit.date_stopped IS NULL AND  visit.date_started < '" + formattedDate + "'", true);
```

Comment:
@slubwama why not use visit.date_started < CURR_DATE() so that you get all unopened dates

Reply:
@ssmusoke done with review

Reply:
@slubwama What does that mean?

Reply:
@ssmusoke find the how this is responded to below in my comment

Reply:
@slubwama I do not understand what you mean